snapcraft/commands/validation_sets.py

```diff
@@ -229,6 +229,13 @@ def edit_validation_sets(
             )
             validate_assertion(edited_validation_sets, **kwargs)
             return edited_validation_sets
+        # assertions with warnings can be submitted
+        except errors.SnapcraftAssertionWarning as err:
+            emit.message(f"{err!s}")
```

Comment:
Do we want to do `emit.progress(permanent=True)` here? Or are we intentionally writing to stdout?

Reply:
I think you're right, especially since this isn't a critical error being displayed.  I'll change it to a progress.